apps/fishing-map/features/timebar/TimebarActivityGraph.utils.ts

```diff
@@ -29,12 +36,23 @@ export const getChunksTimeseries = ({
               key === 'frame' || visibleSublayers[parseInt(key)] === true ? value : 0
             return [key, cleanValue]
           })
-        )
+        ) as TimeSeriesFrame
         return {
           ...activeFrameValues,
           date: quantizeOffsetToDate(frameValues.frame, interval).getTime(),
         }
       })
+      if (aggregationOperation === AggregationOperation.Avg) {
```

Comment:
Thanks 👍 
Making a note to check in agg-tile why exactly this is happening and implement a test